client/src/connection_cache.rs

```diff
@@ -253,7 +251,7 @@ mod tests {
     #[test]
     fn test_connection_with_specified_client_endpoint() {
         // Start a response receiver:
-        let (response_recv_socket, response_recv_exit, keypair2, response_recv_ip) = server_args();
+        let (response_recv_socket, response_recv_exit, keypair2, _) = server_args();
```

Comment:
If it is not used anywhere, should `server_args()` be updated not to return `response_recv_ip`?

Reply:
Fixed. Thanks!